source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -764,6 +765,28 @@ message TransformationAddDeadContinue {
 
 }
 
+message TransformationAddEarlyTerminatorWrapper {
+
+  // Adds a function to the module containing a single block with a single non-
+  // label instruction that is either OpKill, OpUnreachable, or
+  // OpTerminateInvocation.  The purpose of this is to allow such instructions
+  // to be subsequently replaced with wrapper functions, which can then enable
+  // transformations (such as inlining) that are hard in the direct presence
+  // of these instructions.
+
+  // Fresh id for the function
+  uint32 function_fresh_id = 1;
+
+  // Fresh id for the single basic block in the function
+  uint32 label_fresh_id = 2;
```

Comment:
Not a big issue but don't forget to use dot `.`.

Reply:
Done.